doc/guide/guide-bloch.rst

```diff
@@ -407,7 +407,7 @@ An example of generating images for generating an animation outside of Python is
 
 Generating an animation using ffmpeg (for example) is fairly simple::
 
-   ffmpeg -r 20 -b 1800 -i bloch_%01d.png bloch.mp4
+   ffmpeg -i temp/bloch_%01d.png bloch.mp4
```

Comment:
I like this -- it's simple and produces a reasonably nice video. This is a QuTiP tutorial, not an ffmpeg tutorial, so +1 from me.

Reply:
This is the actual only fix that's needed - if you expand the diff, you can see that line 406 saves the files in a temporary directory, but presumably people who were having errors just didn't notice.  The `-r 20` flag is one way of setting fps=20 in this form, so it's not really necessary, and `-b` is setting the bitrate, which here is better controlled coarsely by setting the image resolutions and qualities.